src/components/backend-ai-webui.ts

```diff
@@ -388,7 +388,7 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
       this.is_superadmin = false;
     }
     this._refreshUserInfoPanel();
-    this._writeRecentProjectGroup(this.current_group);
+    globalThis.backendaiclient.utils._writeRecentProjectGroup(this.current_group);
```

Comment:
@lizable you SHOULD NOT CHANGE THIS PART since it is general SDK, not WebUI-only. global.backendaioptions only exists in WebUI.